package.json

```diff
@@ -40,14 +39,19 @@
     "yargs": "^15.3.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.8.4",
+    "@babel/preset-env": "^7.9.6",
+    "@babel/register": "^7.9.0",
     "babel-cli": "^6.26.0",
```

Comment:
I'm now wondering whether @babel/cli and babel-cli are the same thing, just in different versions :/